kubernetes/net-tests/net-test_ds.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Why a DaemonSet?

Reply:
Because it must be installed on all minions

Reply:
in this use case we want to have a pod in each node, so that we can quickly make the different tests

Reply:
Fair.